modules/insights-operator-downloading-archive.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+//
+// * support/remote_health_monitoring/using-insights-operator.adoc
+
+:_content-type: PROCEDURE
+[id="insights-operator-downloading-archive_{context}"]
+= Downloading your Insights Operator Archive
+
+Insights Operator stores gathered data in an archive located in the `openshift-insights` namespace of your cluster. You can download and review the data that is gathered by the Insights Operator. 
+
+.Prerequisites
+
+* Access to the cluster as a user with the `cluster-admin` role.
+
+.Procedure
+
+. Find the name of the currently running pod for the Insights Operator:
++
+[source,terminal]
+----
+$ INSIGHTS_OPERATOR_POD=$(oc get pods --namespace=openshift-insights -o custom-columns=:metadata.name --no-headers  --field-selector=status.phase=Running)
```

Comment:
You could consider removing the environment variable aspect from this procedure, to simplify the commands. For example, you could update this line to the following:

`$ oc get pods --namespace=openshift-insights -o custom-columns=:metadata.name --no-headers  --field-selector=status.phase=Running`

Then, you could update the subsequent step to use a replaceable value and add a call out to highlight that it refers to the name of the pod that was output from the preceding command:

```
[source,terminal]
----
$ oc cp openshift-insights/<insights_operator_pod_name>:/var/lib/insights-operator ./insights-data <1>
----
<1> Replace `<insights_operator_pod_name>` with the pod name output from the preceding command.
```